### PR TITLE
English translation of documentation and documents

### PR DESCRIPTION
**Pull request by benedektarsoly**

### CODE REVIEW

**Review by cserhatit — APPROVED** [review]

**Review by czenczl — APPROVED** [review]

**Review by bucherarnold — APPROVED** [review]

File: .github/workflows/manual_release.yml [hunk 1]
```diff
@@ -37,34 +37,34 @@ jobs:
...
+        # decode the base64 encoded private key and insert it
         run: |
           mkdir -p ~/.gnupg/
           echo "$GPG_SIGNING_KEY" | base64 --decode > ~/.gnupg/private.key
           gpg --import ~/.gnupg/private.key
       - name: Install SSH key
         uses: shimataro/ssh-key-action@v2
         with:
-          # A tartalma egy PEM formatumos (kezdete: '-----BEGIN RSA PRIVATE KEY-----') SSH kulcs
+          # The content is a SSH key with PEM format (start with '-----BEGIN RSA PRIVATE KEY-----')
```
Comment (yoursoft) [inline .github/workflows/manual_release.yml:58]: # The content is an SSH key with PEM format (starts with

Comment (lutischan-ferenc) [inline .github/workflows/manual_release.yml:58]: # The content is an SSH key with PEM format (starts with

File: .github/workflows/manual_release.yml [hunk 2]
```diff
@@ -37,34 +37,34 @@ jobs:
...
+    # Permission check https://github.community/t/who-has-permission-to-workflow-dispatch/133981
 #    if: contains('["rombow","czenczl"]', github.actor)
 
     runs-on: ubuntu-latest
 
     steps:
       - name: Import GPG key
         env:
-          # GPG privat kulcs, ami armor-al exportalva (kezdete: '-----BEGIN PGP PRIVATE KEY BLOCK-----') ez az egesz tartalom base64 elkodolva, az betoltve a secrets tartalomnak
+          # GPG private key exported with armor (start: '-----BEGIN PGP PRIVATE KEY BLOCK-----') this is the whole content base64 encoded, inserted as the secrets content
```
Comment (lutischan-ferenc) [inline .github/workflows/manual_release.yml:48]: starts with

File: .github/workflows/manual_release.yml [hunk 3]
```diff
@@ -37,34 +37,34 @@ jobs:
...
           GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY_GITHUB_TECHUSER }}
-        # dekodoljuk a base64 elkodolt privat kulcsot es betoltjuk
+        # decode the base64 encoded private key and insert it
```
Comment (lutischan-ferenc) [inline .github/workflows/manual_release.yml:50]: and import it?

**Pull request merged**